src/Controller/EnMarche/CoordinatorController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche;
+
+use AppBundle\Controller\CanaryControllerTrait;
+use AppBundle\Entity\Committee;
+use AppBundle\Exception\CommitteeException;
+use AppBundle\Committee\Filter\CommitteeFilters;
+use AppBundle\Form\CoordinatorCommitteeType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * @Route("/espace-coordinateur")
+ * @Security("is_granted('ROLE_COORDINATOR')")
+ */
+class CoordinatorController extends Controller
+{
+    use CanaryControllerTrait;
+
+    /**
+     * @Route("/comites", name="app_coordinator_committees")
+     * @Method("GET")
+     */
+    public function committeesAction(Request $request): Response
+    {
+        $this->disableInProduction();
+
+         try {
+            $filters = CommitteeFilters::fromQueryString($request);
+        } catch (CommitteeException $e) {
+            throw new BadRequestHttpException('Unexpected committee request status in the query string.', $e);
+        }
+
+        $committees = $this->get('app.committee.manager')->getCoordinatorCommittees($this->getUser(), $filters);
+
+        $forms = [];
+        foreach ($committees as $committee) {
+            $form = $this->createForm(CoordinatorCommitteeType::class, $committee, [
+                'action' => $this->generateUrl('app_coordinator_committee_validate', [
+                    'uuid' => $committee->getUuid(),
+                    'slug' => $committee->getSlug(),
+                ]),
+            ]);
+            $forms[$committee->getId()] = $form->createView();
+        }
+
+        return $this->render('coordinator/committees.html.twig', [
+            'results_count' =>  count($committees),
+            'results' => $committees,
+            'forms' => $forms,
+            'filters' => $filters,
+        ]);
+    }
+
+    /**
+     * Pre-approves or pre-refuses a committee.
+     *
+     * @Route("/{uuid}/{slug}/pre-valider-comite", name="app_coordinator_committee_validate")
+     * @Method("POST")
+     */
+    public function validatAction(Request $request, Committee $committee): Response
+    {
+        $form = $this->createForm(CoordinatorCommitteeType::class, $committee);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            try {
+                if ($form->get('refuse')->isClicked()) {
+                    $this->get('app.committee.authority')->preRefuse($committee);
+                    $this->addFlash('info', sprintf('Le comité « %s » a été refusé avec succès.', $committee->getName()));
```

Comment:
shouldn't the flash message be `Le comité « %s » a été pré-refusé avec succès.`

same question about the approval flash message.